### ISSUE TITLE
Feature request: update linux-raw-sys to 0.6.5

### ISSUE THREAD

**Issue by k0tran** [pop-os/cosmic-files#409]
New linux-raw-sys version adds loongarch64 support. Without it compiling cosmic-files on loongarch64 is not possible.

**Comment by KodeBarista** [comment]
`cosmic-files` now seems to pulls in `linux-raw-sys v0.6.5` through `drm-sys`, which should provide the required LoongArch64 support. However, this still needs confirmation from an actual LoongArch64 build.

<img height="500" alt="Image" src="https://github.com/user-attachments/assets/b5ab5033-3c43-49a5-8b30-755145fc41a8" />

**Comment by k0tran** [comment]
Yeah, kinda forgot about this issue. There is no problems with building cosmic-files recent versions. At least right now. Closing this issue.
